src/main/kotlin/com/sourcegraph/cody/ignore/CommandPanelIgnoreBanner.kt

```diff
@@ -27,9 +27,7 @@ class CommandPanelIgnoreBanner() : NonOpaquePanel() {
 
     // These colors cribbed from EditorComposite, createTopBottomSideBorder
     val scheme = EditorColorsManager.getInstance().globalScheme
-    val borderColor =
-        scheme.getColor(EditorColors.SEPARATOR_ABOVE_COLOR)
```

Comment:
@dominiccooney Is there a reason it was implemented this way in the first place??
Code suggests that `scheme.getColor(EditorColors.TEARLINE_COLOR)` may not be sometimes present, and if that the case I'm not convinced that fix is proper?
If you are sure this is ok you can ignore me, just double checking.